src/applications/disability-benefits/all-claims/tests/utils/addToxicExposure.unit.spec.jsx

```diff
@@ -0,0 +1,305 @@
+import { expect } from 'chai';
+import cloneDeep from 'platform/utilities/data/cloneDeep';
+import { addToxicExposure } from '../../utils/submit';
+
+describe('addToxicExposure', () => {
+  describe('when user has no toxic exposure data', () => {
+    it('should return the form data unchanged', () => {
+      const formData = {
+        newDisabilities: [{ condition: 'Chronic Bronchitis' }],
+        veteranInformation: { name: 'John Doe' },
+      };
+
+      const result = addToxicExposure(formData);
+
+      expect(result).to.deep.equal(formData);
+    });
+  });
+
+  describe('when user has toxic exposure data but no conditions', () => {
+    it('should return the form data unchanged', () => {
+      const formData = {
+        newDisabilities: [{ condition: 'Chronic Bronchitis' }],
+        toxicExposure: {
+          gulfWar1990: { bahrain: true },
+          herbicide: { vietnam: true },
+        },
+      };
+
+      const result = addToxicExposure(formData);
+
+      expect(result).to.deep.equal(formData);
+    });
+  });
+
+  describe('when user has toxic exposure conditions selected (not "none")', () => {
+    it('should return all toxic exposure data unchanged', () => {
+      const formData = {
+        newDisabilities: [{ condition: 'Chronic Bronchitis' }],
+        toxicExposure: {
+          conditions: {
+            chronicbronchitis: true,
+            asthma: true,
+          },
+          gulfWar1990: { bahrain: true },
+          gulfWar1990Details: {
+            bahrain: {
+              startDate: '1991-01-01',
+              endDate: '1991-12-31',
+            },
+          },
+          herbicide: { vietnam: true },
+          herbicideDetails: {
+            vietnam: {
+              startDate: '1968-01-01',
+              endDate: '1970-01-01',
+            },
+          },
+          additionalExposures: { asbestos: true },
+          specifyOtherExposures: 'Lead exposure',
+        },
+      };
+
+      const result = addToxicExposure(formData);
+
+      expect(result).to.deep.equal(formData);
+    });
+  });
+
+  describe('when user has mixed conditions including "none"', () => {
```

Comment:
I don't think that this is possible, given the current UI validations. It is not possible to select a condition AND "none".

Reply:
Removed this test case.